pkg/kubelet/kubelet_network.go

```diff
@@ -121,6 +122,64 @@ func (kl *Kubelet) parseResolvConf(reader io.Reader) (nameservers []string, sear
 	return parseResolvConf(reader, scrubber)
 }
 
+func appendAndContinue(composedSearch []string, domain string) bool {
```

Comment:
can you name this in a way that doesn't suggest it is doing the append?  `wouldFit`  or something